tests/prepare/feature/data/check-fips.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+# XXX: How to get the provision method (i.e. the type of the guest)
+# if [[ $PROVISION_METHOD == "container" || $PROVISION_METHOD == "local" ]]; then
+#     echo "Provision method \"$PROVISION_METHOD\" is unsupported."
+#     exit 0
+# fi
```

Comment:
In terms of what @psss described in the meeting, this test would not need to worry about the provisioning method. The upper-level test, `main.fmf` + `test.sh` would take care of it. Depending on `PROVISION_METHOD` envvar, it would skip testing the FIPS feature, therefore `check-fips.sh` would never get called for methods that do not support it.

Reply:
Hi @happz, currently the `main.fmf` supports both `container` and `virtual` when `how == full`. e.g.

```yaml
     4	environment:
     5	    PROVISION_METHODS: container virtual connect local
     6	adjust:
     7	  - when: trigger == commit
     8	    environment:
     9	        PROVISION_METHODS: container
    10	  - when: how == full
    11	    environment:
    12	        PROVISION_METHODS: container virtual
```

So looks we have to take care of `container` when running `tmt -vvv run -a plan --name '/fips/enabled'  provision --how $method`.  

Anything I misunderstood?

Reply:
Follow https://github.com/teemtee/tmt/pull/2558 :) @psss did not merge those changes yet, but once he does, we should see clearer picture on how to follow up with changes to this PR.

Reply:
OK. Let's support FIPS in an individual PR later.